### ISSUE TITLE
 Imperium - Astra Militarum: Profile summaries not working

### ISSUE THREAD

**Issue by BSDataAnon** [BSData/wh40k#5013]
**File:** Imperium - Astra Militarum.catz

**BattleScribe version:** 2.02.02

**Platform:** iPhone / iPod / iPad

**Dropbox:** No

**Description:** Profile summary no longer works. They only appear in inline mode.

**Comment by fat-bob** [comment]
I raised the original bug report. I'll add some supporting information that may help:

1) Data sources and app are up to data (Warhammer 40,000 v1.3.01, 2.02.02 respectively)
2)When view roster is selected for Astra Militarium the profile summary table does not appear at the bottom
3) The output setup seems correct as when I repeat the process for Death Guard it appears as it should.
4) In all cases the profile table will appear embedded under each unit  when "inline" is selected
5) I have tested this on two IOS devices and Android and get the same problem

The attached picture should help.

Thanks
[Profile table problem.pdf](https://github.com/BSData/wh40k/files/3134191/Profile.table.problem.pdf)
